lib/simpleofflinecache.js

```diff
@@ -18,25 +44,25 @@ SimpleOfflineCache.prototype.onFetch = function soc_onFetch(request, response) {
   }
 
   var clone = request.clone();
+  var self = this;
```

Comment:
I don't minde, actually i replaced it with `that`, then I wonder why it was working and was just a scope trivial thing, so I keep it.
But as you commented perhaps is better to avoid any possible confusion is better to change it.